utils.go

```diff
@@ -1,8 +1,41 @@
 package main
 
-import "strings"
+import (
+	"sort"
+	"strings"
+)
 
 func encodeDoubleQuote(inStr string) (outStr string) {
 	outStr = strings.ReplaceAll(inStr, "\"", "\\\"")
 	return
 }
+
+var packageList = map[string]bool{"widget": true, "theme": true}
```

Comment:
This is just a map of `string` to `true`, can it not just be a slice of `string` instead?
I can't see anywhere that the `bool` is `false`.

Reply:
p.s. I don't think that theme is always used...

Reply:
Having a map simplified the process by not having any redundant addition of the same package. If I already have a theme or widget in the packages variable, it won't add a duplicate if `packages` is a map, whereas it would add the duplicate if it was a slice and We will be required to have some kind of duplication removal code again. No matter how many times I assign it in the map, the result will have only one instance of the child - no duplicates.

It seemed that having a map would simplify all of this - hence the map. The boolean will go away in future and will make it an int to track the number of times any method from the said package was called. This would help us in removing the package if a widget was ever removed from the canvas (of course, we haven't added the remove widget func yet).

Reply:
> p.s. I don't think that theme is always used...

But it is. We have an icon on the canvas by default. That required theme right?

Reply:
But the user can remove that icon from the UI... so theme is then not required.
I suspect if you track the import count and remove it if no longer used like you describe above then this initial state would work. However I think getting the list when we export is much simpler (the other comment thread)